src/aggregate.ts

```diff
@@ -121,14 +144,32 @@ abstract class AggregateField {
     return new Count();
   }
 
+  /**
+   * Gets a copy of the Sum aggregate field.
+   *
+   * @returns {Sum}
+   */
+  static sum(property: string): Sum {
+    return new Sum(property);
+  }
+
+  /**
+   * Gets a copy of the Average aggregate field.
+   *
+   * @returns {Average}
+   */
+  static average(property: string): Average {
+    return new Average(property);
+  }
+
   /**
    * Gets a copy of the Count aggregate field.
    *
    * @param {string} alias The label used in the results to describe this
    * aggregate field when a query is run.
    * @returns {AggregateField}
    */
-  alias(alias: string): AggregateField {
+  alias(alias?: string): AggregateField {
```

Comment:
can we add some additional testing on Count now that this is optional?

Reply:
Yes. Added in commit called "Added tests for when an empty alias is provided".